.claude/PRPs/issues/issue-978.md

```
@@ -0,0 +1,538 @@
+  return await serveCommand({ port: servePort, downloadOnly });
+}
+```
+
+**Change 4** — Add `--port` and `--download-only` to `parseArgs` options:
+```typescript
+port: { type: 'string' },
+'download-only': { type: 'boolean', default: false },
+```
+
+**Change 5** — Update `printUsage()` to include `serve`:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)